citations-tool/tool/src/webapp/js/edit_nested_citations.js

```diff
@@ -351,7 +351,7 @@
                 isValidTarget: function (item, container) {
                     var sectiontype = item.data('sectiontype');
                     if (sectiontype=='CITATION'){
-                        if (container.target.hasClass("h4NestedLevel")){
+                        if (container.target.hasClass("h4NestedLevel") || container.target.hasClass("h3NestedLevel")|| container.target.hasClass("h2NestedLevel")){
```

Comment:
Is it worth having a class of something like `holdCitations` on elements that can contain them?